components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5577,12 +5583,19 @@ private boolean isComplexAttribute(String name) {
         }
     }
 
-    private boolean isBooleanAttribute(String name) {
+    private boolean isBooleanAttribute(String name, boolean isCustomScemaAttr) {
 
+        if (isCustomScemaAttr) {
+            return false;
+        }
         return "active".equals(name);
     }
 
-    private boolean isMultivaluedAttribute(String name) {
+    private boolean isMultivaluedAttribute(String name, boolean isCustomScemaAttr) {
+
+        if (isCustomScemaAttr) {
```

Comment:
This logic can be decided even before calling this method. Isn't it ? There is no inhouse login for this decision. WDYT ?

Reply:
Fix with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/459/commits/b5eaab3e1944238b0c0282a38f332f22509a18f5